documentation/blog/2023-06-29-svelte-dev-overhaul.md

```diff
@@ -0,0 +1,98 @@
+---
+title: 'svelte.dev: A complete overhaul'
+description: 'The new site comes with accessibility fixes, new features and bottom navbar'
+author: Puru Vijay
+authorURL: https://puruvj.dev
+---
+
+When the initial version of Svelte v3 was released four years ago, it included the single-page documentation that folks are familiar with. That documentation structure has stayed the same as Svelte's API surface has been increasing steadily and more details were added. As a result, the single page got larger and larger to the point where it was becoming difficult to find things. The community has been asking for a revamp, and now it's here!
+
+Meet the new svelte.dev — a complete overhaul of the old website.
```

Comment:
```suggestion
Meet the new [svelte.dev](https://svelte.dev) — a complete overhaul of the old website.
```

Reply:
Why do we need this? they are already reading on svelte.dev 😅

Reply:
more just for consistency of display I was thinking it might make sense to show them as links and it might make it pop a little more